package-lock.json

```diff
@@ -639,7 +639,7 @@
     },
     "deep-eql": {
       "version": "3.0.1",
-      "resolved": "https://registry.npmjs.org/deep-eql/-/deep-eql-3.0.1.tgz",
+      "resolved": "https://sf-npm.mediarithmics.com/repository/npm-mics/deep-eql/-/deep-eql-3.0.1.tgz",
```

Comment:
Not good => this isn't available for Users not working for mediarithmics (private repo).

Can you fix the `package-lock.json` (hint: check your `.npmrc`, I think that you have to comment some lines before doing a new `npm install`)